content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -31,7 +31,8 @@ <h3>Kubernetes Deployments</h3>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
                 To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
                 how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
-                master schedules mentioned application instances onto individual Nodes in the cluster.
+                master schedules the application instances included in that Deployment to run on individual Nodes in the
```

Comment:
Can you please make a minor change as below 
'master schedules the application instances specified in that Deployment to run on individual Nodes in the'

Reply:
@brentcklein, you could also change “master” to “control plane” if you're confident that helps learners understand things better.

Reply:
Will do. Should “control plane” be capitalized?

Reply:
> Should “control plane” be capitalized?

I think it's fine either way.

Reply:
@brentcklein any update? It'd be great to get those tweaks in if you can. If you're short on time, we can merge this and log a new issue to track the tweaking.